src/main/resources/application.yml

```diff
@@ -66,6 +66,11 @@ jobs:
     # Runs at 21:35, 01:35 and 05:35.
     # Running outside business hours to avoid delivery info suddenly disappearing.
     cron: "0 35 1,5,21 * * *"
+  remove-vca-final-retention-period-data:
+    name: "RemoveVcaFinalRetentionPeriodData"
+    # Runs at 21:45, 01:45 and 05:45.
+    # Running outside business hours to avoid certificate data suddenly disappearing.
+    cron: "0 45 1,5,21 * * *"
```

Comment:
Should this only run once a year on the first of July?

Reply:
That would make more sense, or at least to run it far less often 😄 

However, I was wondering if we want to tie the job's scheduled to the current legislation so tightly? Or if there was ever a use case to set the date manually. Personally I think once a day is ok as there will be so little overhead, but granted not 3 times.